tests/core/test_i64.in

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
```

Comment:
FWIW I'm kind of ambivalent about making the list of tests be explicit (e.g. in a data file or directly in the python) vs having it just be based on the list of files.